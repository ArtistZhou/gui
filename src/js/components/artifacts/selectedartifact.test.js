import React from 'react';
import renderer from 'react-test-renderer';
import SelectedArtifact from './selectedartifact';

it('renders correctly', () => {
  const tree = renderer.create(<SelectedArtifact artifact={{ description: 'text' }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
