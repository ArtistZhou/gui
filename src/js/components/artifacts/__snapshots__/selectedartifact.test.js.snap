// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="muted"
>
  <div
    style={
      Object {
        "display": "flex",
        "fontSize": "12px",
        "justifyContent": "space-between",
        "paddingBottom": "10px",
      }
    }
  >
    <div
      className="MuiFormControl-root-1 list-item list-item-large"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-filled-20 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-shrink-12"
        data-shrink={true}
        htmlFor="artifact-description"
      >
        Description
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-disabled-39 MuiInput-disabled-26 MuiInputBase-formControl-37 MuiInput-formControl-24 MuiInputBase-adornedEnd-41"
        onClick={[Function]}
        style={
          Object {
            "color": "#404041",
            "fontSize": "13px",
          }
        }
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiInput-input-31 MuiInputBase-disabled-39 MuiInput-disabled-26 MuiInputBase-inputAdornedEnd-52"
          disabled={true}
          id="artifact-description"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          required={false}
          type="text"
          value="text"
        />
        <div
          className="MuiInputAdornment-root-53 MuiInputAdornment-positionEnd-56"
        >
          <button
            className="MuiButtonBase-root-64 MuiIconButton-root-58"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "color": "rgba(0, 0, 0, 0.54)",
                "marginBottom": "10px",
              }
            }
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiIconButton-label-63"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root-67"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 0 0-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                />
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
              </svg>
            </span>
            <span
              className="MuiTouchRipple-root-144"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      className="MuiFormControl-root-1 list-item"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-shrink-12"
        data-shrink={true}
        htmlFor="artifact-signed"
      >
        Signed
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-disabled-39 MuiInput-disabled-26 MuiInputBase-formControl-37 MuiInput-formControl-24 MuiInputBase-adornedStart-40"
        onClick={[Function]}
        style={
          Object {
            "color": "#404041",
            "fontSize": "13px",
          }
        }
      >
        <div
          className="MuiInputAdornment-root-53 MuiInputAdornment-positionStart-55"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67 red"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
              opacity=".87"
            />
            <path
              d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
            />
            <path
              d="M15.59 7L12 10.59 8.41 7 7 8.41 10.59 12 7 15.59 8.41 17 12 13.41 15.59 17 17 15.59 13.41 12 17 8.41z"
            />
          </svg>
        </div>
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiInput-input-31 MuiInputBase-disabled-39 MuiInput-disabled-26 MuiInputBase-inputAdornedStart-51"
          disabled={true}
          id="artifact-signed"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root-80 MuiPaper-elevation1-83 MuiExpansionPanel-root-76"
    style={
      Object {
        "background": "#e9e9e9",
        "borderTop": "none",
        "margin": "30px 0",
        "padding": "0 15px",
      }
    }
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-64 MuiExpansionPanelSummary-root-107"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      style={
        Object {
          "padding": 0,
        }
      }
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-111"
      >
        <p>
          Artifact contents
        </p>
        <div
          style={
            Object {
              "marginLeft": "auto",
            }
          }
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-67"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-113"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-115"
      >
        <div
          className="MuiCollapse-wrapperInner-116"
        >
          <div
            className="MuiExpansionPanelDetails-root-117"
            style={
              Object {
                "padding": 0,
              }
            }
          >
            <div />
          </div>
        </div>
      </div>
    </div>
  </div>
  <button
    className="MuiButtonBase-root-64 MuiButtonBase-disabled-65 MuiButton-root-118 MuiButton-text-120 MuiButton-flat-123 MuiButton-disabled-138"
    disabled={true}
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="-1"
    target="_blank"
    type="button"
  >
    <span
      className="MuiButton-label-119"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-67 buttonLabelIcon"
        focusable="false"
        role="presentation"
        style={
          Object {
            "transform": "rotateZ(90deg)",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M10.09 15.59L11.5 17l5-5-5-5-1.41 1.41L12.67 11H3v2h9.67l-2.58 2.59zM19 3H5c-1.11 0-2 .9-2 2v4h2V5h14v14H5v-4H3v4c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
        />
      </svg>
      Download Artifact
    </span>
  </button>
  <div
    className="margin-left inline"
  >
    <button
      className="MuiButtonBase-root-64 MuiButton-root-118 MuiButton-text-120 MuiButton-flat-123"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-119"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-67 red auth buttonLabelIcon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
        Remove this Artifact?
      </span>
      <span
        className="MuiTouchRipple-root-144"
      />
    </button>
  </div>
</div>
`;
